test/common/testlib.py

```diff
@@ -80,6 +80,7 @@
     'Error',
 
     'sit',
+    'todo',
```

Comment:
This would conceptually fit better next to the various `skip*` decorators in the block above.

Reply:
Fixed.